content/snippets/capacity-planning.md

```diff
@@ -1,33 +1,47 @@
 - Servicing transaction reporting information to clients
 - Maintaining varying amounts of historical data
 
-Capacity planning for `rippled` involves two factors:
+How you meet your `rippled` capacity planning needs depends on how you address these technical factors:
 
 - The [configuration settings](#configuration-settings) that affect resource utilization
-- The [network and hardware] (#network and hardware) requirements to achieve consistent, good performance across the XRP Ledger network
+- The [network and hardware](#network-and-hardware) requirements to achieve consistent, good performance across the XRP Ledger network
 
 ## Configuration Settings
 
 Ripple recommends the following guidelines to improve performance. You can set the following parameters in the `rippled.cfg` file to improve performance for your `rippled` server.
 
 ### Node Size
 
-The `node_size` parameter determines the size of database caches. Ripple recommends you always set it to `huge` for production servers. Increased cache size requires less disk I/O and allows `rippled` to improve performance. The trade-off to improving performance better is that memory requirements increase.
+The `node_size` parameter determines the size of database caches. Larger database caches decrease disk I/O requirements at a cost of higher memory requirements. Ripple recommends you always use the largest database cache your available memory can support. See the following table for recommended settings.
+
+#### Recommendation
+
+| Available RAM for `rippled` | `node_size` value | Notes                              |
+|:----------------------------|:------------------|:-----------------------------------|
+| < 8GB                       | `tiny`            | Not recommended                    |
+| 8GB                         | `low`             |                                    |
+| 16GB                        | `medium`          |                                    |
+| 32GB                        | `huge`            | Recommended for production servers |
 
 ### Node DB Type
 
-The `type` field in the `node_db` section of the `rippled.cfg` file sets the type of key-value store that `rippled` uses to persist the XRP Ledger in the Ledger Store. You can set the value to either `rocksdb` or `nudb`. For setting the store that `rippled` uses to persist history shards, if you are participating in history sharding, see [History Sharding](XREF: concept-history-sharding.md#shard-store-configuration).
+The `type` field in the `node_db` section of the `rippled.cfg` file sets the type of key-value store that `rippled` uses to persist the XRP Ledger in the Ledger Store. You can set the value to either `rocksdb` or `nudb`.
+
+`rippled` offers a history sharding feature that allows you to store a randomized range of ledgers in a separate shard store. For more information about how to use this feature, see [History Sharding](XREF: concept-history-sharding.md#shard-store-configuration).
```

Comment:
This paragraph could use something to link it to this section. Maybe something like this after the first sentence?

> You may want to configure the shard store to use a different type of key-value store than the ledger store.